.eslintignore

```diff
@@ -0,0 +1,31 @@
+tests/*
+
+src/middleware.js
+src/plugin.template.js
+
+.DS_Store
+node_modules
+/dist
+cypress.env.json
+/tests/e2e/videos/
```

Comment:
remove things that do not apply.  No tests, cypress ect.

Reply:
Done